### ISSUE TITLE
About the costom dataset

### ISSUE THREAD

**Issue by vegetableclean** [yongsongH/AIDSRGAN-MICCAI2022#6]
Hello, my name is Chieh, and I am a master's student from Taiwan and currently being a research assistant in the school. First of all, I want to express my gratitude for your outstanding work in designing the model for joint denoising and super-resolution. After reading your papers, I have some ideas to implement it for my remote sensing project. However, I am a bit confused about the need for three input datasets. Most of the papers or codes I have studied so far typically involve only two inputs, one for low resolution (LW), which we down-sampled from the high resolution and the other for ground truth (GT). Is it possible to train your code using only two inputs? Thank you.

**Comment by yongsongH** [comment]
Hi, Chieh! Thank you for your attention!

As you can see, our work can be followed by three different types of input.
If you want to give less input, the data loaders and loss function should be modified.

I hope this helps you.🍻

**Comment by vegetableclean** [comment]
Dear, thank you for your response. I have an idea that I would like to inquire about. Since I just want to try your performance on my dataset, would you recommend that I first use your provided codes (PSRGAN) for fine-tuning or training? If so, do you have any suggested steps or recommendations?

**Comment by vegetableclean** [comment]
Dear Mr. Huang, I apologize for bothering you again. As I am relatively new to this field, I was wondering if I could seek your advice. May I ask you about the approximate training time for GANs on IR images? I have an RTX 3080 10G graphics card, and I'm curious about how long it typically takes to train a GAN model.

**Comment by yongsongH** [comment]
Hello!

Thank you for your attention to our work.
More training details and other information can be found at this [link](https://github.com/yongsongH/Infrared_Image_SR_PSRGAN/issues?q=is%3Aissue+is%3Aclosed).
I hope this will be helpful to you.

BTW, it would be our pleasure if our work could be cited!
